lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -78,6 +78,65 @@ def approved?
           end
         end
 
+        def merge!(options = {})
+
+          # fallback to default merge process if user "forces" merge
+          if(options[:skip_lgtm])
+            super options
+          else
+            say "Merging pull request ##{self.number}: '#{self.title}', from '#{self.feature_branch_name}' into '#{self.base_branch_name}'"
+
+            options[:base] ||= 'master'
+            feature_branch_name = GitReflow.current_branch
```

Comment:
Ok. This logic for `options[:base]` will be gone when we remove the `base` option altogether.